quetz/tests/api/test_main_packages.py

```diff
@@ -85,9 +95,12 @@ def test_delete_package_versions_with_package(
 
     assert len(versions) == 0
 
-    files = pkgstore.list_files(public_channel.name)
+    for filename in package_filenames:
+        init_files.remove(filename)
```

Comment:
this should not happen when the subdir is `noarch` because otherwise the client thinks that the channel does not exist.

Reply:
I'm not sure to understand.
This code removes the package filenames from the list of channel files (`repodata.json`, `index.html`...), to compare the files list before an after removing a package (only the package archive should have been removed).

Reply:
Sorry, I thought that you were deleting empty repodata.json files. I should've looked more carefully.

Reply:
> Clean repodata.json from empty platform when deleting a package

That was the commit message that confused me :)

Reply:
No problem, thanks for review